helm/thingsboard/templates/mqtt-transport.yaml

```
@@ -103,6 +103,10 @@ spec:
           - name: MQTT_BIND_PORT
             value: "{{ .Values.mqtt.port.number }}"
           {{- end }}
+          {{- range .Values.mqtt.extraEnvVars }}
+          - name: {{ .key }}
```

Does it make sense to keep environment variables in uppercase?
```suggestion
          - name: {{ .key | upper }}
```

---

I didn't thought about it, but I suppose it makes sense